nativelib/src/main/scala/scala/scalanative/unsafe/CFuncPtr.scala.gyb

```diff
@@ -10,7 +10,11 @@
 package scala.scalanative
 package unsafe
 
-sealed trait CFuncPtr
+import scala.scalanative.runtime.Boxes
+
+sealed trait CFuncPtr {
+ def toPtr: Ptr[Byte] = Boxes.boxToPtr[Byte](Boxes.unboxToCFuncRawPtr(this))
```

Comment:
I am a bit wary of adding custom members to this trait which is a bit special. Consider defining `Ptr.cFuncPtrToPtr` instead.